charts/templates/clusterrolebinding.yaml

```diff
@@ -33,3 +33,20 @@ subjects:
   name: {{ include "litmus.fullname" . }}-admin
   namespace: {{ .Release.Namespace }}
 {{ end }}
+{{ if .Values.litmusPortal.enabled }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: litmus-rb
```

Comment:
Should it be `name: litmus`?
Ref: https://mayadata-team.slack.com/archives/C010QVBA10A/p1597491206079500

Reply:
I take refer from our litmus-portal manifest and try to sync with this: https://github.com/litmuschaos/litmus/blob/master/litmus-portal/k8s-manifest.yml#L169